### PR TITLE
fix(server): mount OS storage as virtio by default

### LINKED ISSUE #146: Disk gets mounted as IDE not VirtIO by default
Creating a server with the CLI (from a template) results in the OS disk being mounted in `IDE` mode instead of `VirtIo` mode. This has big performance impacts. It's also not the default the UpCloud hub uses.

### PR DESCRIPTION
**Pull request by Davincible**
Fix #146

### CODE REVIEW

**Comment by kangasta** [comment]
Thank you for the contribution!

Could you also edit the create tests expected request to check for `virtio` address instead of empty string?

File: internal/commands/server/create.go [hunk 1]
```diff
@@ -97,6 +97,7 @@ func (s *createParams) processParams(planSvc service.Plans, storageSvc service.S
 
 		s.StorageDevices = append(s.StorageDevices, request.CreateServerStorageDevice{
 			Action:  "clone",
+			Address: "virtio:0",
```
[the remark this answers is not in the document]
Comment (Davincible) [reply internal/commands/server/create.go:100]: I wasn't sure whether to put it in, as it doesn't matter. I went with it because I thought it'd be better to be explicit, but I'll remove it.

**Review by kangasta — APPROVED** [review]
